corehq/ex-submodules/dimagi/utils/django/email.py

```diff
@@ -42,6 +42,7 @@ def mark_local_bounced_email(bounced_addresses, message_id):
         except MessagingSubEvent.DoesNotExist:
             pass
         else:
+            metrics_counter('commcare.messaging.email.bounced', tags={'domain': subevent.parent.domain})
```

Comment:
keep in mind that multiple emails can be present in `bounced_addresses`, so we might want to use `len` of that in updating our metrics. additionally, this will only track bounces from messaging events (which i guess is what we want anyway) but not bounces from other emails triggered by the system

Reply:
Great pointer, thank you. I'll do exactly that.